contents/docs/apps/user-interview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: User Interviewer
+github: https://github.com/PostHog/user-interview-app
+installUrl: https://app.posthog.com/project/apps?name=user-interview
+thumbnail: ../../apps/thumbnails/user-interview.png
+tags:
+    - user-interview
+---
+
+### What does the User Interviewer app do?
+
+This app extends functionality for [the GeoIP Entricher app](/apps/geoip-enrichment), offering the ability to automatically discard unwanted information in a way which is helpful users with privacy or compliance needs.  
+
+### What are the requirements for this app?
+
+This app requires either PostHog Cloud, or a self-hosted PostHog instance running [version 1.30.0](https://posthog.com/blog/the-posthog-array-1-30-0) or later.
+
+Not running 1.30.0? Find out [how to update your self-hosted PostHog deployment](https://posthog.com/docs/runbook/upgrading-posthog)!
+
+You'll also need to have [the GeoIP Entricher app](/apps/geoip-enrichment) installed. 
+
+### How do I install this app?
+
+1. Make sure you have enabled `opt_in_site_apps: true` in your posthog-js config.
+2. Install the app from the PostHog App Repository
+3. Customize the text and theme using the app config
+4. Configure the app by creating a feature flag (see below)
+
+That's it!
+
+### Is the source code for this app available?
+
+PostHog is open-source and so are all apps on the platform. The [source code for this app](https://github.com/posthog/user-interview-app) is available on GitHub.
+
+### How do I configure the app?
+
+1. Create a feature flag to control who sees it. For this example, we'll use the name `user-interview`
+2. Set up the filters for the people that you want to speak to. You can create a filter based on properties, such as location or email, but not  events. If you want to invite users based on them doing certain actions you'll need to update the users property in the app once they've done that action.
+3. Additionally, if you want to invite users based on historical actions, create an insight for that action and then use that to create a static cohort.
+4. Set the filter `Seen User Interview Invitation - {featureFlagName}` to `is not set` so that it doesn't show to users who have seen the user interview already. (This property will be added once the user has interacted with the popup - either to close it or to book in a time)
```

Comment:
```suggestion
4. Set the filter `Seen User Interview Invitation - {FEATURE_FLAG_NAME}` to `is not set` so that it doesn't show to users who have seen the user interview already. This property is added once the user has interacted with the popup - either to close it or to book in a time.
```